deploy.gradle

```diff
@@ -191,12 +191,18 @@ task deploy {
 
                 def user = execute "whoami"
                 def host = execute "hostname -s"
+                def filesPath = System.getenv("FILES_DIR_PATH")
```

Comment:
why enviornment variable? can we simple not hardcode in each job config  as we other task param. I mean this is not a secret so IMO the simpler and easier is the best